### PR TITLE
SGM-6030 - Sourcegraph App page

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Description

implementation of the app landing page from the [design mockup](https://www.figma.com/file/ExXmC7SLGQld8c1ZzVD4we/App-Landing-Page?node-id=11%3A1382&t=POTgjsvfISg9Gkkj-1)

## Ref

[SG Issue](https://github.com/sourcegraph/about/issues/6030)
[Gitstart Ticket](https://clients.gitstart.com/sourcegraph/1/tickets/SGM-6030)

https://user-images.githubusercontent.com/45232708/223743900-268f8cf9-23ec-44f3-9eab-b1faa7231ce9.mov

### CODE REVIEW

**Comment by fabicastp** [comment]
Design feedback: 
- All body copy (text-lg or text-base) should be gray-200, not white.
- Starting at the 1022px width breakpoint, the 3-col section should all remain left-aligned in their containers. As is, the icons and body copy are switching to center-aligned and it looks weird 
cc @traceyljohnson

**Comment by gitstart-sourcegraph** [comment]
Thanks @fabicastp @slimsag,
Requested changes has been addressed. Kindly take a look.

**Comment by slimsag** [comment]
@fabicastp is right; I edit'd the wrong one, my bad! I'll let you fix @gitstart-sourcegraph

File: src/pages/app.tsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                        </div>
+                    ),
+                    content: (
+                        <div className="mx-0.5 flex flex-col px-4">
+                            <ol className="m-0 text-left">
+                                <li className="ml-2.5 text-xs leading-7">
+                                    <div className="self-start text-sm leading-7">
+                                        Install via{' '}
+                                        <Link
+                                            href="https://storage.googleapis.com/sourcegraph-app-releases/2023.03.23+204957.f499bb/sourcegraph_2023.03.23+204957.f499bb_linux_amd64.deb"
```
Comment (slimsag) [inline src/pages/app.tsx:162]:
```suggestion
                                            href="https://brew.sh"
```

File: src/pages/app.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+                        </div>
+                    ),
+                    content: (
+                        <div className="mx-0.5 flex flex-col px-4">
+                            <ol className="m-0 text-left">
+                                <li className="ml-2.5 text-xs leading-7">
+                                    <div className="self-start text-sm leading-7">
+                                        Download the{' '}
+                                        <Link
+                                            href="https://brew.sh"
```
Comment (slimsag) [inline src/pages/app.tsx:199]:
```suggestion
                                            href="https://storage.googleapis.com/sourcegraph-app-releases/2023.03.23+205301.ca3646/sourcegraph_2023.03.23+205301.ca3646_linux_amd64.deb"
```

**Comment by slimsag** [comment]
I went ahead and fixed the links, this is good to go from my POV.

**Review by taylorsperry — APPROVED** [review]

**Pull request merged**